### PR TITLE
Include libspectre and ghostscript

### PR DESCRIPTION
**Pull request by aleixpol**

### CODE REVIEW

**Comment by tsdgeos** [comment]
bot, build

**Comment by aleixpol** [comment]
> You're trying to build spectre with cmake, it doesn't have cmake
Yes, I don't know what I was thinking...

File: org.kde.okular.json [hunk 1]
```diff
@@ -204,6 +204,32 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://libspectre.freedesktop.org/releases/libspectre-0.2.8.tar.gz",
+                    "sha256": "65256af389823bbc4ee4d25bfd1cc19023ffc29ae9f9677f2d200fa6e98bc7a8"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "ghostscript",
```
[the remark this answers is not in the document]
Comment (nedrichards) [reply org.kde.okular.json]: Ah, no worries. There's loads of old tarballs that were uploaded before 64bit arm processors existed and so refuse to configure correctly on that architecture. We need to remove the old configure and then reconfigure using the tooling in the SDK which is new enough to understand about the archtiecture. An example of this is SDL1: https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json you rm-configure:true and then attach the script at the bottom. It should work fine then.

**Comment by nedrichards** [comment]
@aleixpol it was ghostscript that failed on aarch64, not spectre

**Comment by nedrichards** [comment]
bot, build

**Pull request merged**